api/v1alpha1/oadp_types.go

```diff
@@ -190,10 +190,12 @@ type DataMover struct {
 
 // MoverConfig defines configurations for VolSync options
 type VolumeOptions struct {
-	// storageClassName can be used to override the StorageClass of the source PVC
+	// storageClassName can be used to override the StorageClass of the source
```

Comment:
Can we have different storage class used for source and destination? They might both be using the same driver but different deletionPolicy for example.

Reply:
@shubham-pampattiwar wdyt of this? ^ I think then we'd need to do the same for each of the new types for backup and restore, which would bring us to a lot of config values.

Reply:
@eemcmullan @kaovilai yeah IMO it would make sense. Lets create a struct something like this:
```
type DatamoverVolumeOptions struct {
    SrcVolumeOptions VolumeOptions
    DestinationVolumeOptions VolumeOptions
}
```